desktop_version/src/Script.cpp

```diff
@@ -519,9 +520,13 @@ void scriptclass::run(void)
                 textx = ss_toi(words[2]);
                 texty = ss_toi(words[3]);
 
+                // If words[4] ends with L, set large to true.
+                textlarge = (words[4].size() > 0 && words[4][words[4].size() - 1] == 'l');
```

Comment:
Use `endsWith`. Then the code becomes self-documenting and the comment is unnecessary.

Also, check for `L` too, just in case someone does `text,,,,26L`, which wouldn't lowercase the last argument due to not ending with an argument separator (this doesn't happen with `say,26L` because the simplified parser passes the argument through with an ending argument separator).
```suggestion
                textlarge = endsWith(words[4].c_str(), "l") || endsWith(words[4].c_str(), "L");
```